wheel/python/clvm_rs/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .eval_error import EvalError  # noqa: F401
+from .program import Program  # noqa: F401
```

Comment:
is the point of this to re-export these names? If so, I believe the way do to that (to not get the warnings) is to add:

```
__all__ = ["Program", "EvalError"]
```

Reply:
Sweet trick!